sys-libs/pam/metadata.xml

```diff
@@ -12,6 +12,12 @@
 		flag will create a PAM module that links to the Berkeley DB (as
 		provided by <pkg>sys-libs/db</pkg>) installed in /usr/lib and
 		will thus not work for boot-critical services authentication.
+	</flag>
+	<flag name="xcrypt">
```

Comment:
well I am not sure it will eventually be merged by glibc, + this information is not really important

Reply:
To verify I understand correctly: you are saying you are not sure glibc will ever remove libxcrypt from its codebase? That is in contradiction with current glibc dev plans...

I can simplify this message to just the first sentence just in case glibc changes it's mind :)

Reply:
@10ne1  yes, that would be nice